include/fastdds/dds/topic/IContentFilterFactory.hpp

```diff
@@ -45,6 +72,15 @@ struct IContentFilterFactory
             const ParameterSeq& filter_parameters,
             IContentFilter*& filter_instance) = 0;
 
+    /**
+     * Delete an IContentFilter instance.
+     *
+     * @param [in]  filter_class_name   Filter class name for which the factory is being called.
+     *                                  Allows using the same factory for different filter classes.
+     * @param [in]  filter_instance     A pointer to a filter instance previously returned by create_content_filter.
+     *                                  The factory takes ownership of the filter instance, and can decide to destroy
+     *                                  it or keep it for future use.
+     */
```

Comment:
The `@return` information is missing

Reply:
Commit 1691cb0ad51e26655f7adf750b603bd5b9adbd34